entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+cat >/etc/motd <<EOL
+  _____
+  /  _  \ __________ _________   ____
+ /  /_\  \\___   /  |  \_  __ \_/ __ \
+/    |    \/    /|  |  /|  | \/\  ___/
+\____|__  /_____ \____/ |__|    \___  >
+        \/      \/                  \/
+A P P   S E R V I C E   O N   L I N U X
+Documentation: http://aka.ms/webapp-linux
+NodeJS quickstart: https://aka.ms/golang-qs
```

Comment:
Should be "Go quickstart"?

Reply:
yes. and we need the link to point also to some documetation as with node. Ill follow up with Josh

---
In reply to: [166155099](https://github.com/Azure-App-Service/go/pull/1#discussion_r166155099) [](ancestors = 166155099)